manifest.json

```diff
@@ -1,13 +1,14 @@
 {
   "manifest_version": 2,
   "name": "MEGA",
-  "description": "Secure Cloud Storage and Chat",
-  "version": "109101.103.97",
+  "description": "__MSG_extDesc__",
+  "version": "1.99.99",
```

Comment:
Why the version change? that will break / prevent the extension from running unpacked / in development mode.

Reply:
```suggestion
  "version": "109101.103.97",
```